Src/Web/Web.Shared.Net/ApplicationInsightsHttpModule.cs

```diff
@@ -153,6 +156,7 @@ private void AddCorreleationHeaderOnSendRequestHeaders(HttpApplication httpAppli
             }
         }
 
+#if NET40
```

Comment:
why wouldn't we do the same for NET45? We can restore the context just in case, can we? Or ASP.NET module will do it guaranteed?